src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -7,15 +7,15 @@ import AnonymousMessageRepo from '../../../service/repo/anonymousMessageRepo';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
 import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
-import {ANON_PM, GROUP_INFO} from '../constant';
+import {ANON_PM, CHAT_ATOM, GROUP_INFO} from '../constant';
 import {ChannelList} from '../../../../types/database/schema/ChannelList.types';
 import {ChannelTypeEnum} from '../../../../types/repo/SignedMessageRepo/SignedPostNotificationData';
 import {Context} from '../../../context';
 import {PostNotificationChannelList} from '../../../../types/database/schema/PostNotificationChannelList.types';
 import {convertTopicNameToTopicPageScreenParam} from '../../../utils/string/StringUtils';
 
-const chatAtom = atom({
-  key: 'chatAtom',
+export const chatAtom = atom({
+  key: CHAT_ATOM,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded string to a constant for the `key` in `chatAtom` is a good practice as it reduces the risk of typos and makes the code more maintainable. However, please ensure that the `CHAT_ATOM` constant is defined and exported correctly in the `constant` module.
```diff
-  key: 'chatAtom',
+  key: CHAT_ATOM,
```
Also, the export of `chatAtom` has been added which wasn't there before. This is fine as long as it's intended to be used elsewhere in the application. If not, it might be unnecessary to export it.

Lastly, the import statement on line 10 has been modified to include `ANON_PM` and `CHAT_ATOM`. Ensure these are being used in the file or else remove them to avoid unused variable warnings.
```diff
- import {ANON_PM, GROUP_INFO} from '../constant';
+ import {ANON_PM, CHAT_ATOM, GROUP_INFO} from '../constant';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->